src/web/routes/application/middleware/configure-get.test.js

```diff
@@ -0,0 +1,53 @@
+const test = require('tape')
+const { getPreviousPage } = require('./configure-get')
+
+test('getPreviousPage() should return undefined for first page', (t) => {
+  const steps = [
+    {
+      path: 'first',
+      next: 'second'
+    },
+    {
+      path: 'second',
+      next: 'third'
+    }
+  ]
+
+  const result = getPreviousPage(steps, steps[0])
+
+  t.equal(result, undefined)
+  t.end()
+})
+
+test('getPreviousPage() should return path of previous page when not the first page', (t) => {
+  const steps = [
+    {
+      path: 'first',
+      next: 'second'
+    },
+    {
+      path: 'second',
+      next: 'third'
+    }
+  ]
+
+  const result = getPreviousPage(steps, steps[1])
+
+  t.equal(result, 'first')
+  t.end()
+})
+
+test('getPreviousPage() should throw an error when the step is not in the list of steps', (t) => {
+  const steps = [
```

Comment:
This repeated 3 times in this file, let's have it as a constant.

Reply:
Done.